data/json/professions.json

```diff
@@ -1062,8 +1063,11 @@
   {
     "type": "profession",
     "id": "football_player",
-    "name": "Football Player",
-    "description": "Your epic touchdown was ruined when your coach's guts were torn out by undead fans.  Now you're first pick in the apocalypse draft.",
+    "name": "Soccer Player",
+    "description": {
+      "male": "Soccer, rugby — whatever you term it, you'd played it extensively before the Cataclysm and were quite the sportsman.  Now, your trade has left you with a set of football armour that you'd packed along during the evacuation, a strong throwing arm, and experience in pitch-side tussling.  With your ball at your side, the only keepsake you'd been able to hold on to through the end of days, you suppose that it's time to hustle.",
+      "female": "Soccer, rugby — whatever you term it, you'd played it extensively before the Cataclysm and were quite the sportswoman.  Now, your trade has left you with a set of football armour that you'd packed along during the evacuation, a strong throwing arm, and experience in pitch-side tussling.  With your ball at your side, the only keepsake you'd been able to hold on to through the end of days, you suppose that it's time to hustle."
```

Comment:
I don't think this is quite what @TheShadowFerret had in mind with their review, rather than eliminating any mention of US football, you should have eliminated any mention of **other sports** that were not US football, since the professions is clearly representing an American football player, not a soccer player or a rugby one.